nucliadb/nucliadb/tests/integration/test_purge.py

```diff
@@ -205,9 +205,13 @@ async def test_purge_orphan_shard_detection(
     node = random.choice(available_nodes)
     orphan_shard = await node.new_shard(
         kbid="deleted-kb",
-        similarity=utils_pb2.VectorSimilarity.COSINE,
         release_channel=utils_pb2.ReleaseChannel.STABLE,
-        normalize_vectors=False,
+        config=nodewriter_pb2.VectorIndexConfig(
```

Comment:
I know right now we don't have config other than the vector index config, but just in case: should we name the param `vector_index_config`?